src/module-elasticsuite-core/view/frontend/web/js/form-mini.js

```diff
@@ -30,7 +30,6 @@ define([
     $.widget('smileEs.quickSearch', $.mage.quickSearch, {
         options: {
             autocomplete: 'off',
-            minSearchLength: 2,
```

Comment:
I think this could be kept, for me this is acting as a fallback value, if we inject something else via the template it should override this "2".

Reply:
But it should be tested

Reply:
agreed